pkg/liqoctl/install/utils.go

```diff
@@ -0,0 +1,75 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	helm "github.com/mittwald/go-helm-client"
+	"gopkg.in/yaml.v3"
+	"helm.sh/helm/v3/pkg/action"
+	"k8s.io/client-go/rest"
+
+	helm2 "github.com/liqotech/liqo/pkg/liqoctl/install/helm"
+	"github.com/liqotech/liqo/pkg/liqoctl/install/kubeadm"
+	"github.com/liqotech/liqo/pkg/liqoctl/install/provider"
+	"github.com/liqotech/liqo/pkg/utils"
+)
+
+func getProviderInstance(providerType string) provider.InstallProviderInterface {
+	if providerType == "kubeadm" {
+		return kubeadm.NewKubeadmProviderConstructor()
+	}
+	return nil
+}
+
+func initHelmClient() (*helm.HelmClient, *rest.Config, error) {
+	kubeconfigPath, ok := os.LookupEnv("KUBECONFIG")
+	if !ok {
+		kubeconfigPath = filepath.Join(os.Getenv("HOME"), ".kube", "config")
+	}
+
+	config, err := utils.UserConfig(kubeconfigPath)
+	if err != nil {
+		fmt.Printf("Unable to create client config: %s", err)
+		return nil, nil, err
+	}
+	helmClient, err := helm2.InitializeHelmClientWithRepo(config)
+	if err != nil {
+		fmt.Printf("Unable to create helmClient: %s", err)
+		return nil, nil, err
+	}
+	return helmClient, config, nil
+}
+
+func installOrUpdate(ctx context.Context, helmClient *helm.HelmClient, k provider.InstallProviderInterface) error {
+	output, _, err := helm2.GetChart(helm2.LiqoChartFullName, &action.ChartPathOptions{}, helmClient.Settings)
+	if err != nil {
+		return err
+	}
+
+	k.UpdateChartValues(output.Values)
+
+	raw, err := yaml.Marshal(&output.Values)
+	if err != nil {
+		return err
+	}
+	chartSpec := helm.ChartSpec{
+		// todo (palexster): Check if it ReleaseName and LiqoNamespace are really configurable
```

Comment:
Does this TODO still apply?

Reply:
Yes, unfortunately we never tested Liqo in a namespace that is not `liqo`. Liqo 0.2.X surely did not work.